src/app/components/FormElements/TimeSlider.tsx

```diff
@@ -60,7 +90,7 @@ export default class TimeSlider extends React.Component<TimeSliderProps> {
     }
 
     return (
-      <div style={{margin: 5}}>
+      <div style={{ margin: 5 }}>
         <Range
```

Comment:
Have you tried this out with the tool tips?  Was just looking at the docs and it looks super easy to add it in:

```ts
// At the top near your imports
const createSliderWithTooltip = Slider.createSliderWithTooltip;
const Range = createSliderWithTooltip(Slider.Range);
// Then just use Range how you are using it.
```

Reply:
No I haven't explored this yet, but agree that it . could be helpful